meinberlin/apps/projectcontainers/forms.py

```diff
@@ -6,27 +6,35 @@
 
 from . import models
 
+LABELS = {
+    'name': _('Title of your container'),
+    'description': _('Short description of your container'),
+    'information': _('Description of your container'),
```

Comment:
the information label should be set in a separate dictionary directly on the information form.